src/main/java/Duke.java

```diff
@@ -1,10 +1,55 @@
+import command.Command;
+import exceptions.DukeException;
+import storage.Storage;
+import tasklist.TaskList;
+import ui.UI;
+import utility.Parser;
+
+/**
+ * Main Duke class.
+ */
 public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private UI ui;
+
+    /**
+     * Constructor.
+     * @param filePath Path where disk file should be created.
+     */
```

Comment:
I think there should be a line between the description and the parameter section of your JavaDocs.